### PR TITLE
Changes

### PR DESCRIPTION
**Pull request by NeboCode**
- Folder structure revamp to keep the root directory cleaner.
- Scripts are moved from head section to the bottom of body in order to increase performance.
- Cleaned up obsolete files (favicon.ico in root directory but there being no references to it in code).
- Minified your CSS code to increase page speed.

I did not want to update your icons, however as an alternative icon set you might consider entirely free Material Icons:
https://material.io/resources/icons/?style=baseline

I was a long time FontAwesome user however I found MaterialIcons to be genuinely faster and better quality over FA.

Thank you and have a nice day.

### CODE REVIEW

**Comment by metzgegu** [comment]
Thank you for your contribution, Nice Pull request 🥇 

About your changes :
- I'm not a fan of minified css because it make it more difficult to maintain and for a personal website the improvement will not be really noticeable.

Yes I should give Material Icons a try some day. 👍 

Please update your PR according to current develop changes.

**Comment by NeboCode** [comment]
Updated

File: resources/css/style.css [hunk 1]
```diff
@@ -0,0 +1 @@
+.container{display:flex}.container__info{max-width:80%;font-family:monospace;text-align:center;margin:0 auto}.container__title{font-size:5rem;margin-top:50px}.container__description{margin:2%;font-size:2.5rem}.container__social{color:#000;margin:2%;font-size:5rem}.container__calendar{margin-top:50px}.wday{margin-right:5px}i{font-size:2.3rem}i:hover{color:#595959}.contrib-legend{display:none}.activity-overview-box{display:none}#readme{padding:5px;margin-top:40px}
```
Comment (metzgegu) [inline resources/css/style.css]: I think we should keep this file not minimized, or can you explain me why it is really need (apart of the small speed increase) ?

File: index.html [hunk 2]
```diff
@@ -50,7 +38,9 @@ <h1 class="container__title">Guillaume Metzger</h1>
                 <a class="container__social" href="mailto:Guillaume.metzger5@gmail.com" target="_blank"><i class="fas fa-at"></i></a>
             </div>
         </div>
-	    
+    script src="https://cdnjs.cloudflare.com/ajax/libs/markdown-it/11.0.0/markdown-it.min.js" integrity="sha512-0tDdhRf9dcDufaaOQQdvywK56y+cwomy6rjOrqk05N61cQOTdO2Cqqgq8EfXVBDXJe0TH9smqvDicQzMiBG14g==" crossorigin="anonymous"></script>
```
Comment (PierreBerger) [inline index.html:41]:
```suggestion
    <script src="https://cdnjs.cloudflare.com/ajax/libs/markdown-it/11.0.0/markdown-it.min.js" integrity="sha512-0tDdhRf9dcDufaaOQQdvywK56y+cwomy6rjOrqk05N61cQOTdO2Cqqgq8EfXVBDXJe0TH9smqvDicQzMiBG14g==" crossorigin="anonymous"></script>
```

**Comment by NeboCode** [comment]
Unminified.

**Pull request merged**